rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/NodeSourcePolicyFactory.java

```diff
@@ -92,7 +99,7 @@ public static NodeSourcePolicy create(String policyClassName, String infrastruct
         // initializing parameters
         BooleanWrapper result = stub.configure(policyParameters);
         if (!result.getBooleanValue()) {
-            throw new RuntimeException("Cannot configure the policy " + policyClassName);
+            throw new RuntimeException("Cannot configure the policy " + policy.getClass().getSimpleName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)